lib/store.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
```

Comment:
More documentation in general, I know you left some comments explaining it on gh comments, but for possible contributors to read will be interesting.